CHANGES.md

```diff
@@ -8,6 +8,7 @@ Change Log
 * Deprecated
     * Deprecated `GroundPrimitive.geometryInstance`. It will be removed in 1.20. Use `GroundPrimitive.geometryInstances` instead.
     * Deprecated `TileMapServiceImageryProvider`. It will be removed in 1.20. Use `createTileMapServiceImageryProvider` instead.
+    * Deprecated `throttleRequests` flag in `TerrainProvider`. It will be removed in 1.19. It is replaced by an optional `Request` object that stores information used to prioritize requests.
```

Comment:
I just noticed that `Request` is still private.  Perhaps let's not deprecate this until we make the `RequestScheduler` public.  If you agree, update the 3D Tiles roadmap.